### ISSUE TITLE
Example: sketch01.xml-elements in v0.4

### ISSUE THREAD

**Issue by chikashimiyama** [j45ch/sptdf04_issues#44]
The relationship between handles and points is ambiguous in pointset.
This need to be clarified in the specification by itself.

**Comment by j45ch** [comment]
I agree that the formulation shoud be made unambiduous n the soecs, I attempted to do thisnow: plz review.

concerning the example `sketch01.xml` "how the handles relates to the points."
I don't understand how the handle relationship to a point needs to be defined; the sequence of points and handles in the pointset determines that relationship.

@Chikashi: I know that you attach handles on either side of the points in Zirkonium, but when doing the blending in a spline the handles actually belong to the two neighboring points. Additionally there are no handles beyond the first and the last anchors.

I added the mention in the specs that anchor points are shared between segments of multisegment cubic-bezier curves.
